cloudwatch_event_rules.tf

```diff
@@ -23,9 +23,9 @@ resource "aws_cloudwatch_event_rule" "lambda" {
 }
 
 resource "aws_cloudwatch_event_target" "lambda" {
-  for_each = var.cloudwatch_event_rules
-
-  arn   = lookup(each.value, "cloudwatch_event_target_arn", local.function_arn)
-  rule  = aws_cloudwatch_event_rule.lambda[each.key].name
-  input = lookup(each.value, "cloudwatch_event_target_input", null)
+  for_each       = var.cloudwatch_event_rules
+  event_bus_name = lookup(each.value, "event_bus_name", null)
```

Comment:
please add a blank line after the `for_each`

Reply:
Done!